modules/simple_bucket/variables.tf

```diff
@@ -78,6 +78,14 @@ variable "retention_policy" {
   default = null
 }
 
+variable "custom_placement_config" {
+  description = "Configuration of the bucket's custom location in a dual-region bucket setup. If the bucket is designated a single or multi-region, the variable are null."
+  type = object({
```

Comment:
Is this syntax supported in TF 0.13?  

https://github.com/terraform-google-modules/terraform-google-cloud-storage/blob/master/modules/simple_bucket/versions.tf#L18